lib/cli/package.json

```diff
@@ -37,5 +37,8 @@
   "devDependencies": {
     "@types/shelljs": "^0.8.7",
     "typescript": "3.9.3"
+  },
+  "peerDependencies": {
```

Comment:
this one needs to be in the other package.json, I assume:
`lib/cli/tooling/package.json`